builtin/settingtypes.txt

```diff
@@ -623,6 +623,13 @@ update_last_checked (Last update check) string
 #    Ex: 5.5.0 is 005005000
 update_last_known (Last known version update) int 0
 
+#    Type of occlusion_culler
+occlusion_culler (Occlusion Culler) enum old old,new
```

Comment:
Fewer settings the better, can we delete the old culler?

Reply:
We will delete it in a month.